src/main/java/frc/robot/tuning/stateMachineTuner.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.tuning;
+
+import frc.robot.RobotMap;
+import frc.robot.drive.Drive;
+import frc.robot.gamepieces.GamePieceController;
+import frc.robot.logging.RobotLogManager;
+import frc.robot.sensors.Gyrometer;
+
+import org.apache.logging.log4j.Logger;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class stateMachineTuner implements Tuner {
```

Comment:
Class names should always start with a Capital letter. Also, the name is not meaningful. What does this actually tune? Rename it.